src/helpers/mdObjBucketHelper.ts

```diff
@@ -20,3 +20,12 @@ export function getBucketItems(
 export function isBucketEmpty(buckets: VisualizationObject.IBucket[], bucketName: string): boolean {
     return isEmpty(getBucketItems(buckets, bucketName));
 }
+
+export function getIdentifierFromBucketsItem(
+    bucket: VisualizationObject.IBucket[],
+    bucketLocalIdentifier: string,
+    type: string,
```

Comment:
The `type` parameter seems confusing. You need to read the method content to find out what it does. I would just call the method `getMeasuresLocalIdentifiers(buckets, bucketLocalIdentifier)` and rewrite its content to the following:

```
return getBucketItems(bucket, bucketLocalIdentifier)
	.filter(item => VisualizationObject.isMeasure)
    .map(item => item.localIdentifier);
```

Reply:
This function I define it returns localIdentifier by type (attribute/measure/view) it's not only for type is measure so if someone else want to get localIdentifier by type they can reuse it

Reply:
I'm not sure it works like this. It uses visualization object bucket item, i.e., it is `measure` or `visualizationAttribute`. It will not work with `attribute` or `view`. It seems it confused even you which proves my point :) I don't like the usage of arbitrary strings such as `type` when you don't know what are possible values. It is not defined as constant or enum or another type. You can get around it by the means I suggested or come up with something else.